staging/src/k8s.io/kube-aggregator/pkg/apis/apiregistration/helpers.go

```diff
@@ -86,9 +87,16 @@ func (s ByVersionPriority) Less(i, j int) bool {
 
 // APIServiceNameToGroupVersion returns the GroupVersion for a given apiServiceName.  The name
 // must be valid, but any object you get back from an informer will be valid.
-func APIServiceNameToGroupVersion(apiServiceName string) schema.GroupVersion {
+func APIServiceNameToGroupVersionTenant(tenantedApiServiceName string) (string, string, schema.GroupVersion) {
```

Comment:
likely need a unit test if there's not one yet

Reply:
Yep, added.